### PR TITLE
Fix: Update styles for archive campaign modal UI

### PR DESCRIPTION
**Pull request by jdghinson**
## Description

This PR updates the archive campaign modal UI to match the designs in Zeplin. I also added hover states to both the cancel and archive campaign button. Updated our design system to make good use of the neutral colors.

## Affects

Affects the archive campaign modal in campaigns.

## Visuals

Before
https://www.loom.com/share/53b768c90434437d98e051f201e0043d?sid=33dc9237-5ef4-43e4-9339-fad58f813c99

After
https://www.loom.com/share/293a55f6123b42fdb9a5eb53b1496166?sid=c84b8d6c-94f0-4519-803d-debf2e7bd8ed

## Testing Instructions

1. Open an active campaign
2. Click on the more button and select archive campaign
3. Check the updated modal UI

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [x] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

**Review by JasonTheAdams — COMMENTED** [review]
Love the tweaking, @jdghinson! Left a comment for educational and consideration purposes. 😄

File: src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss [hunk 1]
```diff
@@ -211,12 +212,21 @@ select[name="campaignId"] {
...
 
     .confirmButton {
         border: 1px solid #d92d0b;
         background-color: #d92d0b;
         color: #fff;
+
+        &:hover {
+            border: 1px solid #F2320C;
+            background-color: #F2320C;
+        }
```
[the remark this answers is not in the document]
Comment (jdghinson) [reply src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss:229]: @JasonTheAdams The neutral colors I created for our design system has not been pushed to the main branch.  So I used the hex code for the cancel button.

I'll make the changes for the archive button.

File: src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss [hunk 2]
```diff
@@ -185,38 +185,48 @@ select[name="campaignId"] {
     font-size: 16px;
     font-weight: 500;
     line-height: 1.5;
-    padding: var(--givewp-spacing-3);
+    color: #1f2937;
```
Comment (JasonTheAdams) [inline src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss:188]: @jdghinson I'm guessing we just don't have a design system color we want to use here?

File: src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (JasonTheAdams) [reply src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss]: Looks so good! 🎉

File: src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss (hunk 2 above)
Comment (jdghinson) [reply src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss:188]: Updated

Comment (JasonTheAdams) [reply src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss]: Nice! 🎉 

As a friendly note, @jdghinson, now that you're a dope dev-design unicorn, the convention we use is to include the hash in the follow up comment that is intended to resolve the comment. For example, for this comment you'd say something like:

Resolved in 0fc7b20

😃

**Review by JasonTheAdams — APPROVED** [review]
Fantastic work, @jdghinson! The UI is even more polished the code is even better because you worked on it.

chefs kiss

**Comment by jdghinson** [comment]
Thank you!! @JasonTheAdams

**Review by rickalday — APPROVED** [review]
Passed manual QA tests.

**Pull request merged**